static/app/views/settings/project/loaderScript.spec.tsx

```diff
@@ -0,0 +1,348 @@
+import {
+  render,
+  screen,
+  userEvent,
+  waitForElementToBeRemoved,
+} from 'sentry-test/reactTestingLibrary';
+
+import {t} from 'sentry/locale';
+import LoaderScript from 'sentry/views/settings/project/loaderScript';
+
+describe('LoaderScript', function () {
+  let org, project, projectKeys;
+
+  beforeEach(function () {
+    org = TestStubs.Organization();
+    project = TestStubs.Project();
+    projectKeys = TestStubs.ProjectKeys();
+
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/keys/`,
+      method: 'GET',
+      body: projectKeys,
+    });
```

Comment:
as a personal preference, I usually avoid doing that as I already ran into issues because of it 🤷‍♀️ usually I like to create a separate function for the mocks and call them inside each `it`